doc/manual/manual.tex

```diff
@@ -3174,6 +3174,7 @@ \subsection{Overview}
 total 932
 -rw-rw-r-- 1 bangerth bangerth  11134 Dec 11 10:08 depth_average.gnuplot
 -rw-rw-r-- 1 bangerth bangerth  11294 Dec 11 10:08 log.txt
+-rw-rw-r-- 1 bangerth bangerth     42 Dec 11 10:07 original.prm
```

Comment:
I wonder how you created this file on Wolfgang's machine in the past? :smirk:

Reply:
psst. :wink: